packages/swirl-components/src/components/flip-description-list-item/flip-description-list-item.css

```diff
@@ -0,0 +1,67 @@
+@import "../../styles/media-queries.css";
+
+:host {
+  display: block;
+
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.description-list-item {
+  display: block;
+  max-width: 100%;
+  color: var(--s-text-default);
+  line-height: var(--s-line-height-base);
+}
+
+.description-list-item__term {
+  display: block;
+  margin-bottom: var(--s-space-4);
+  font-weight: var(--s-font-weight-semibold);
+}
+
+.description-list-item__description {
+  display: block;
+  max-width: 40rem;
+  margin: 0;
+}
+
+@container description-list (min-width: 480px) {
+  .description-list-item {
+    display: flex;
+  }
+
+  .description-list-item__term {
+    margin-right: var(--s-space-16);
+    margin-bottom: 0;
```

Comment:
just personal taste but i think it is better to have display options (flex stuff) above the box-model stuff (margin)

Reply:
That's from my current postcss-sort setup. Let's adjust this, once we have figured out a shared sort config.

Reply:
👍